menpo/math/decomposition.py

```diff
@@ -96,10 +166,10 @@ def pca(X, centre=True, inplace=False, eps=1e-10):
     if d < n:
         # compute covariance matrix
         # C (covariance): d x d
-        C = np.dot(X.T, X) / (n - 1)
+        C = np.dot(X.conj().T, X) / (n - 1)
```

Comment:
these are fixes for complex PCA right? Maybe add them to the PR description? I take it we don't break behavior for existing matrices with these changes?